gc/base/HeapMap.cpp

```diff
@@ -214,7 +214,7 @@ MM_HeapMap::numberBitsInRange(MM_EnvironmentBase *env, void *lowAddress, void *h
 	uintptr_t baseIndex, topIndex;
 	
 	/* Validate passed heap references */
-	Assert_MM_true(lowAddress < highAddress);
+	Assert_MM_true(lowAddress <= highAddress);
```

Comment:
With this change a part (Subspace) of the heap can be. When Nursery is unified (100% tilted) prior to compact, the other part of Nursery is literately 0%. If we happen to clear mark map during percolate global on the empty part we hit this assertion. 

An alternative is to keep this assertion on strict '<' and prevent even an attempt of clearing the mark map on empty subspace. I did try that but did not find an elegant way to do it. I can try again...